apps/wpcom-block-editor/src/wpcom/editor.js

```diff
@@ -1,7 +1,25 @@
+/* eslint-disable import/no-extraneous-dependencies */
+
+/**
+ * WordPress dependencies
+ */
+
+import { registerPlugin } from '@wordpress/plugins';
+
+/* eslint-enable import/no-extraneous-dependencies */
+
 /**
  * Internal dependencies
  */
 import './features/deprecate-coblocks-buttons';
 import './features/fix-block-invalidation-errors';
 import './features/reorder-block-categories';
 import './features/tracking';
+
+import InserterMenuTrackingEvent from './features/tracking/wpcom-navigation-menu-search-handler';
```

Comment:
Shouldn't this be in `./features/tracking` and just imported here?

Reply:
In that case, should we register the plugin there as well then, right?